src/icons/style.scss

```diff
@@ -0,0 +1,34 @@
+@import "variables";
+
+@font-face {
+  font-family: '#{$icomoon-font-family}';
+  src:
+    url('#{$icomoon-font-path}/#{$icomoon-font-family}.ttf?f1o68b') format('truetype'),
```

Comment:
@milbau ten hash jest tutaj potrzebny myślisz?

Reply:
It may be needed when we run "docs-serve" but webpack seems to cut ?f1o68b anyway so it doesn't work as it should